doc/man5/config.pod

```diff
@@ -60,22 +70,18 @@ done with the following directive:
 
  .pragma [=] dollarid:value
 
-Where B<value> is one of the following:
-
-=over 4
-
-=item B<off> or B<false>
-
-This is the default behavior. For example, C<foo$bar> is interpreted as
-C<foo> followed by the expansion of the variable C<bar>.
-
-=item B<on> or B<true>
-
-This specifies that dollar signs are part of the symbol name and
+The default behavior, where the B<value> is B<false> or B<off>, is to treat
+the dollarsign as indicating a variable name; C<foo$bar> is interpreted as
+C<foo> followed by the expansion of the variable C<bar>. If B<value> is
+B<true> or B<on>, then C<foo$bar> is a single seven-character name nad
 variable expansions must be specified using braces or parentheses.
-For example, C<foo$bar> is treated as a single seven-character name.
 
-=back
+ .progma [=] includedir:value
```

Comment:
Typo progma

Reply:
You actually did not resolve this one. Sorry for not noticing earlier.

Reply:
strange, because I thought I fixed in when @beldmit noticed it the other day.  oh well.  updated fixup commit pushed yet again :)